app/uk/gov/hmrc/hmrcemailrenderer/templates/fhdds/fhddsBetaSubmissionConfirmation.scala.txt

```diff
@@ -0,0 +1,50 @@
+This email includes all the information you'll need to use the service and give us your feedback.
+
+What the service is for:
+This is a new service by which organisations will apply for approval to be registered as fulfilment businesses.
+Please note: this test version will only allow you to run through the process of registering, it is not the registration process itself, which will only be available from 1st April 2018
+
+It should be used by:
+Whoever in your organisation will be responsible for applying to the register,
+Ideally this person should be the person who will be able to sign a declaration of compliance on behalf of the organisation,
+The information you provide should be legitimate as it will be used to test our technical processes.
+
+Your information will not be maintained into the final service when it launches in April, so you will be asked to provide much of this information again at that time. It is probably advisable to keep a copy of the information you submit on file for future reference.
+
+The facility to recover your username and/or password using information relating to your FHDDS application will not be available during the FHDDS Private Beta. This means that if you lose or forget your username and/or password, there may be significant delay in gaining access to the details of your application while the problem is sorted out.
+
+We therefore recommend that you use the same username and/or password that you already use to access other government services online in relation to your business. You should then be able to use information relating to one of these other services to recover your username and password if necessary.
+
+We expect the application process to be improved before being launched in April. We also expect more information about the due diligence requirements being agreed to by then.
+
+How to start testing:
+This link will take you straight to the test version of the service:
+@{params("verificationLink")}
+
+Please don't share this link with other people.
+We'll be monitoring how the service is used by invitees and may need to restrict access.
+
+How to give us your feedback:
+If you want to comment on any aspect of the service please email John Salisbury  at john.salisbury@@digital.hmrc.gov.uk
+The more feedback you give us, the better we can make the next version of this service.
+
+How long testing lasts:
+You can submit an application any time during the next 5 days, but it would be really helpful to the team if you made a submission on the afternoon of 11th December 2017.
+After that we'll make and test a second version of the service based on the feedback we get.
+
+Why you got this email:
+This email is being sent to people who've been involved in our research so far, or who've volunteered to help us test this service.
+
+Thank you for your help and feedback,
+Karina Immanuel
+Digital Service Manager
+Fulfilment House Due Diligence Scheme
+If you’re unsure an email is from HMRC:
```

Comment:
the "If you're unsure ..." can be done by 
`@{uk.gov.hmrc.hmrcemailrenderer.templates.helpers.txt.template_footer()}`